tests/e2e/specs/front-end-metadata.spec.js

```diff
@@ -33,6 +35,9 @@ describe( 'Front end metadata insertion', () => {
 		await startUpTest();
 		await setSiteKeys();
 		await selectScreenOptions( { recrawl: true, advanced: false } );
+
+		// Reset display name to compare metadata with default values.
+		await setUserDisplayName( 'admin', '' );
```

Comment:
If `filters.spec.js` file ran before this file then some tests throws error because inside `filter.spec.js` file we are changing the display names so here we are resetting it to default to make sure that tests don't fail no matter the order.

Reply:
Great find!